src/site/content/en/authors/individual.njk

```diff
@@ -7,6 +7,7 @@ renderData:
   title: "{{ paged.title | i18n(locale) or title }}"
   description: "{{ paged.description | i18n(locale) or description }}"
   rss: "{{ paged.href }}feed.xml"
+  hero: "{{ paged.data.hero }}"
```

Comment:
Did authors not have hero images before? What does this look like - I'm a bit puzzled here.

Reply:
They did, but when I changed the data structure to use i18n data for authors I forgot to add this to it. So this adds it back.